Tools/ssl/multissltests.py

```diff
@@ -1,12 +1,12 @@
 #!./python
-"""Run Python tests against multiple installations of OpenSSL and LibreSSL
+"""Run Python tests against multiple installations of crypto libraries
```

Comment:
```suggestion
"""Run Python tests against multiple installations of cryptographic libraries.

Currently tested are:

* OpenSSL
* LibreSSL
* AWS-LC
```

Reply:
CPython's `ssl` module links both `libssl` and `libcrypto`, two distinct libraries produced by OpenSSL/LibreSSL/AWS-LC/BoringSSL's respective codebases. Not to bee too persnickety, but using `libcrypto` here doesn't seem quite right.

Reply:
'cryptography' is fine by me